pkg/splunk/enterprise/util.go

```diff
@@ -396,7 +396,7 @@ func getRemoteObjectKey(cr splcommon.MetaObject, appFrameworkConfig *enterpriseA
 	volumePath = volumePath[index+1:]
 	location := appSrc.Location
 
-	remoteObjectKey = volumePath + location + appName
+	remoteObjectKey = filepath.Join(volumePath, location, appName)
```

Comment:
This is another fix as part of CSPL-1515 where if we miss `/` in the volume `path` and/or in the `location` in the appSources, we were not creating correct remoteObjectKey, which would eventually result in download failure.